app/plugin_commands.go

```diff
@@ -150,5 +157,11 @@ func (a *App) tryExecutePluginCommand(args *model.CommandArgs) (*model.Command,
 	}
 
 	response, appErr := pluginHooks.ExecuteCommand(a.PluginContext(), args)
+
+	// Checking if plugin crashed after the command.
+	if err := pluginsEnvironment.PerformHealthCheck(matched.PluginId); err != nil {
+		message := fmt.Sprintf("/%s command crashed the %s plugin. Please contact your system administrator.", trigger, matched.PluginId)
+		return matched.Command, nil, model.NewAppError("ExecutePluginCommand", message, nil, "err= Plugin failed to start: "+matched.PluginId, http.StatusInternalServerError)
```

Comment:
I believe in this scenario, this specific instance of the slash command crashed the plugin right? Maybe an error like `Plugin crashed due to slash command`, and provide some info about the command?